packages/components/src/drop-zone/index.tsx

```diff
@@ -194,7 +143,9 @@ export function DropZoneComponent( {
 
 	return (
 		<div { ...restProps } ref={ ref } className={ classes }>
-			{ isDraggingOverElement && <DropIndicator label={ label } /> }
+			<MaybeFade show={ isDraggingOverElement || false }>
+				<DropIndicator label={ label } />
+			</MaybeFade>
```

Comment:
With the above suggestion to implement reduced motion preference with CSS, we end up not needing to create the entire `MaybeFade` component if we do something like:

```suggestion
			<div
				className={
					isDraggingOverElement
						? 'components-drop-zone__fade-enter'
						: 'components-drop-zone__fade-exit'
				}
			>
				<DropIndicator label={ label } />
			</div>
```

Reply:
Yep, that makes sense 👍🏻